conans/client/new.py

```diff
@@ -0,0 +1,111 @@
+conanfile = """from conans import ConanFile, CMake, tools
+import os
+
+
+class {name}Conan(ConanFile):
+    name = "{name}"
+    version = "{version}"
+    license = "<Put the package license here>"
+    url = "<Package recipe repository url here, for issues about the package"
+    settings = "os", "compiler", "build_type", "arch"
+    options = {{"shared": [True, False]}}
+    default_options = "shared=False"
+    generators = "cmake"
+
+    def source(self):
```

Comment:
I don't like too much these lines uncommented. I just keep a "pass" and the rest of lines commented. build and package could be more useful but just if you are using cmake, so maybe I would prefer these lines commented too and maybe more commented examples like call configure/make